server/tables/tables.go

```diff
@@ -12,6 +12,12 @@ import (
 
 const (
 	mySQLDialect = "mysql"
+
+	
+	// UserGroup membership status constants
+
+	MemberStatus = 0
+	MembershipRequestedStatus = 1
```

Comment:
Can we make this a proto enum instead? I think we should try to keep go-specific stuff out of here and just stick to primitive data types.

Reply:
Done